src/Domain/TagsTimeDependent.hpp

```diff
@@ -0,0 +1,164 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <boost/optional.hpp>
+#include <cstddef>
+#include <memory>
+#include <string>
+#include <tuple>
+#include <unordered_map>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"  // For Tags::Normalized
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/FaceNormal.hpp"
+#include "Domain/FunctionsOfTime/FunctionOfTime.hpp"
+#include "Domain/FunctionsOfTime/Tags.hpp"
+#include "Domain/Tags.hpp"
+#include "Time/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace domain {
+/// \ingroup ComputationalDomainGroup
+/// \brief %Tags for the domain.
+namespace Tags {
+/// The Inertial coordinates, the inverse Jacobian from the Grid to the Inertial
```

Comment:
should there be `\brief` and `\details` to make sure doxygen does what it's supposed to do? (same for below tags)

Reply:
I'm not sure exactly what you mean, but with the `///` style Doxygen comments the line break signals transition from brief to detailed. E.g.
```cpp
/// This will all be part of the brief
///
/// While this will all be part of the details
```
This is not true for the C-style `/*!` Doxygen comments. There a `\brief` is necessary. `\details` is something we added (I'm not sure I like it as much as I used to :man_shrugging: )

Reply:
ah, understood. I had overlooked the difference between the `///` and `/*!`. Thanks!

Reply:
I submitted #2074 to clarify this in our documentation :)